examples/spacy/spacy-tokenizer.py

```diff
@@ -0,0 +1,66 @@
+from pipeline import (
+    Pipeline,
+    PipelineCloud,
+    Variable,
+    pipeline_function,
+    pipeline_model,
+)
+
+
+@pipeline_model
+class model:
+    def __init__(self):
+        self.nlp = None
+
+    @pipeline_function
+    def predict(self, input: str) -> list:
+        doc = self.nlp(input)
+        # (optional) your spacy code here or you can return entire spacy object
+        # to manipulate on a client that has spacy installed.
+        res = []
+        for token in doc:
+            res.append([token.text, token.pos_, token.dep_])
+        return res
```

Comment:
just a style thing so not important, but I quite like using list comprehensions for this kind of thing, e.g.
```python
return [[token.text, token.pos_, token.dep_] for token in doc]